src/components/20-molecules/top-content-bar/README.md

```diff
@@ -21,6 +22,30 @@ If no attribute `href` is set, it will render an `axa-button` and the `onClick`
 
 The attribute `href` specifies the link, that should be opened by pressing on the button with the text defined in `ctatext`. This is mainly for semantic reasons, you still need to forward the user to the new location, by listening to the `onClick` callback. Just providing the `href` attribute will not do anything by itself.
 
+### icon
+
+Using the string-valued attribute `icon` as icon name, an icon will be rendered. To see the full list of possible icons, see the [axa-icon](https://github.com/axa-ch-webhub-cloud/pattern-library/blob/develop/src/components/10-atoms/icon/README.md) README.
+
+### stickymobile
+
+Set the boolean attribute `stickymobile` to `true` to display the bar at the bottom of the screen on mobile devices with width < 411px.
+
+### overlaydesktop
+
+Set the boolean attribute `overlaydesktop` to `true` to display the bar on top of the DOM element that comes after this element. Usually that would be the Keyvisual CTA component. This feature applies only for the screen width larger than 411px. For mobile devices the bar will either be displayed as usually with `position:static` or at the bottom of the screen if the attribute `stickymobile` is enabled.
```

Comment:
Grammar, wording: "This feature applies only **to** non-mobile screen widths." + usually -> usual

Reply:
I removed the "overlaydesktop" totally from the code. From JS, CSS and readme. Only stickymobile remains. If I understood it well, you agreed with Ben that this should be implemented in axa-ch-core CSS.

Reply:
"you agreed with Ben that this should be implemented in axa-ch-core CSS." Yes, that was the recommendation.